metaworld/__init__.py

```diff
@@ -75,6 +75,7 @@ def test_tasks(self) -> List[Task]:
 
 
 def _encode_task(env_name, data):
+    #pickle.dumps() is serializing kwargs data
```

Comment:
why did you add this line?

Reply:
did it while analyzing the code, removed now